src/main/scala/com/fulcrumgenomics/cmdline/FgBioMain.scala

```diff
@@ -85,16 +86,24 @@ class FgBioMain extends LazyLogging {
     htsjdk.samtools.util.Log.setGlobalLogLevel(htsjdk.samtools.util.Log.LogLevel.WARNING)
 
     val startTime = System.currentTimeMillis()
-    val parser    = new CommandLineParser[FgBioTool](name)
-    val exit      = Sopt.parseCommandAndSubCommand[FgBioCommonArgs,FgBioTool](name, args, Sopt.find[FgBioTool](packageList)) match {
+    val parser    = new CommandLineParser[FgBioTool](name)  // Keep a reference to the parser so we can get the command line
+    val exit      = parser.parseCommandAndSubCommand[FgBioCommonArgs](args, Sopt.find[FgBioTool](packageList)) match {
       case Sopt.Failure(usage) =>
         System.err.print(usage())
         1
-      case Sopt.CommandSuccess(cmd) =>
+      case Sopt.CommandSuccess(_) =>
         unreachable("CommandSuccess should never be returned by parseCommandAndSubCommand.")
-      case Sopt.SubcommandSuccess(command, subcommand) =>
+      case Sopt.SubcommandSuccess(_, subcommand) =>
         val name = subcommand.getClass.getSimpleName
         try {
+          parser.commandLine.foreach { commandLine =>
+            subcommand.toolInfo = FgBioToolInfo(
+              name        = name,
+              commandLine = commandLine,
+              description = parser.formatShortDescription(Sopt.inspect(subcommand.getClass).description),
+              version     = this.version
```

Comment:
I think to be safe here (and the other place `version` is used, it should probably be implemented as `subcommand.getClass.getPackage.getImplementationVersion`.  That way if the sub-command is in another project that's using fgbio, it'll get the version for the sub-command, not the version for fgbio.